automated/linux/ml-bench/tvm-bench.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+# shellcheck disable=SC1091
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+#default values
+TVM_WHEEL='https://github.com/tlc-pack/tlcpack/releases/download/v0.7.dev1/tlcpack-0.7.dev1-cp38-cp38-manylinux2010_x86_64.whl'
+TVM_BENCH='https://github.com/tom-gall/tvm-bench.git'
+SKIP_INSTALL=false
+
+while getopts "s:a:d:" o; do
```

Comment:
'd' option is commented out. Is this intentional?

Reply:
Yeah 'd' is intended to be commented out for now. More args will be added in the future.